<style> 

</style>

<table>
    <tr>
      <th>TITLE</th>
      <th>AUTHORS</th> 
      <th>DOI</th>
      <th>DATE</th>
    </tr>
    <tr>
      <td>The Human Pangenome Project: a global resource to map genomic diversity.</td>
      <td>Wang T., Antonacci-Fulton L., Howe K., et al.</td> 
      <td>10.1038/s41586-022-04601-8</td>
      <td>2022-04-20</td>
    </tr>
    <tr>
      <td>Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant classes.</td>
      <td>Ebler J, Ebert P, Clarke WE, Rausch T, Audano PA, Houwaart T, Mao Y, Korbel JO, Eichler EE, Zody MC, Dilthey AT, Marschall T. </td> 
      <td>10.1038/s41588-022- 01043-w</td>
      <td>2022-04-11</td>
    </tr>
    <tr>
      <td> Concerted modification of nucleotides at functional centers of the ribosome revealed by single-molecule RNA modification profiling.   |  Bailey AD, Talkish J, Ding H, Igel H, Duran A, Mantripragada S, Paten B, Ares M.Elife </td>
      <td>Bailey AD, Talkish J, Ding H, Igel H, Duran A, Mantripragada S, Paten B, Ares M.Elife </td> 
      <td>10.7554/eLife.76562</td>
      <td>2022-04-06 </td>
    </tr>
    <tr>
      <td>Segmental duplications and their variation in a complete human genome. </td>
      <td>Vollger MR, Guitart X, Dishuck PC, et al.</td> 
      <td>10.1126/science.abj69 65</td>
      <td>2022-04-01</td>
    </tr>
    <tr>
        <td>StainedGlass: Interactive visualization of massive tandem repeat structures with identity heatmaps.</td>
        <td>Vollger MR, Kerpedjiev P, Phillippy AM, Eichler EE.</td> 
        <td>10.1093/bioinformatics /btac018</td>
        <td>2022-01-10</td>
      </tr>
    <tr>
        <td> The UCSC Genome Browser database: 2022 update. </td>
        <td>Lee BT, Barber GP, Benet-Pag√®s A, et al.</td>
        <td>10.1093/nar/gkab959</td>
        <td>2022-01-07 </td>
    </tr>
    <tr>
        <td>Profiling variable-number tandem repeat variation across populations using repeat-pangenome graphs </td>
        <td>Tsung-Yu Lu, Human Genome Structural Variation Consortium, Mark Chaisson.</td>
        <td>10.1038/s41467-021- 24378-0</td>
        <td>2021-07-12 </td>
    </tr>
    <tr>
        <td>Targeted long-read sequencing identifies missing disease-causing variation</td>
        <td>Miller, D., Sulovari, A., Wang, T., et al.</td>
        <td>10.1016/j.ajhg.2021.06 .006</td>
        <td>2021-07-02 </td>
    </tr>
    <tr>
        <td>lra: A long read aligner for sequences and contigs</td>
        <td>Jingwen Ren, Mark Chaisson. </td>
        <td>10.1371/journal.pcbi.10 09078</td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
    <tr>
        <td></td>
        <td></td>
        <td></td>
        <td></td>
    </tr>
  </table>